GardenApp/app/src/main/java/com/cs1530_group1/gardenapp/Garden.java

```diff
@@ -250,7 +250,9 @@ public static Garden stringToGarden(String g)
     public Species getSpeciesInfo(String speciesName)
     {
     	Species newSpecies = null;
-		
+		Date pruneDate = null;
+		Date plantDate = null;
+    	
 		//go through speciesList looking for the species with the name equal to the parameter
         for (Species s : speciesList)
         {
```

Comment:
Shouldn't pruneDate and plantDate be set to null inside the loop so they get reset for every plant?